client/signup/config/flows-pure.js

```diff
@@ -237,7 +237,15 @@ export function generateFlows( {
 
 	if ( isEnabled( 'signup/wpcc' ) ) {
 		flows.wpcc = {
-			steps: [ 'oauth2-user' ],
+			steps: [ 'oauth2' ],
```

Comment:
Should this remain as `oauth2-user`? There currently isn't a `oauth2` step, so this line causes the normal `/start/wpcc` flow to fail (just loads a grey screen as it can't find the step.